python/mxnet/ndarray/sparse.py

```diff
@@ -417,6 +423,19 @@ def data(self):
         """
         return self._data()
 
+    @indices.setter
```

Comment:
Do we plan to support these setters in the future? Seems scipy does not provide these interfaces. It feels problematic to call an individual setter since this might introduce data inconsistency issue.

Reply:
scipy supports `some_csr.indices = [1,2,3]`

Reply:
I see.